src/doc/rustdoc/src/lints.md

```diff
@@ -274,7 +278,11 @@ warning: unopened HTML tag `script`
 2 | | /// </script>
   | |_____________^
   |
-  = note: `#[warn(invalid_html_tags)]` on by default
+  note: the lint level is defined here
+ --> foo.rs:1:9
+  |
+1 | #![warn(rustdoc::invalid_html_tags)]
+  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
As @joshtriplett noticed, the output was incorrect before: `invalid_html_tags` is allowed by default. This happened because doctests `lints.md` are not tested. He's going to open an issue for testing this output.